src/PublicApiGenerator/EventNameBuilder.cs

```diff
@@ -0,0 +1,62 @@
+using System.CodeDom;
+using Mono.Cecil;
+
+namespace PublicApiGenerator
+{
+    public static class EventNameBuilder
+    {
+        public static string AugmentEventNameWithEventModifierMarkerTemplate(EventDefinition eventDefinition, string name,
+            MemberAttributes addAccessorAttributes, MemberAttributes removeAccessorAttributes)
+        {
+            var addVTableAttributes = (addAccessorAttributes & MemberAttributes.VTableMask);
+            var removeVTableAttributes = (removeAccessorAttributes & MemberAttributes.VTableMask);
+
+            if (eventDefinition.DeclaringType.IsInterface)
+            {
+                if (addVTableAttributes == MemberAttributes.New && removeVTableAttributes == MemberAttributes.New)
+                {
+                    return $"{string.Format(CodeNormalizer.EventModifierMarkerTemplate, $"new{CodeNormalizer.EventRemovePublicMarker}")}{name}";
+                }
+
+                return $"{string.Format(CodeNormalizer.EventModifierMarkerTemplate, CodeNormalizer.EventRemovePublicMarker)}{name}";;
+            }
+
+            var addScopeAttributes = (addAccessorAttributes & MemberAttributes.ScopeMask);
+            var removeScopeAttributes = (removeAccessorAttributes & MemberAttributes.ScopeMask);
+
+            if (addScopeAttributes == MemberAttributes.Static && removeScopeAttributes == MemberAttributes.Static)
+            {
+                return $"{string.Format(CodeNormalizer.EventModifierMarkerTemplate, "static")}{name}";
```

Comment:
I think people recommend to use concatenation whenever interpolation is only used to insert things at the very beginning or end of the string.

Reply:
Done